### PR TITLE
Thoughts after 117

### PR DESCRIPTION
**Pull request by OR13**
A few thoughts after IETF 117.

Goals:

- Address identity up front (by not addressing it directly).
- Frame feeds in the context of the API
- Relate media types to feeds

This PR is mostly placeholders for future work, but hopefully highlights some areas for follow up based on our in person discussions.

### CODE REVIEW

File: draft-birkholz-scitt-scrapi.md
Comment (JAG-UK) [inline draft-birkholz-scitt-scrapi.md:252]: What's the payoff for having a YAML version?

**Review by JAG-UK — COMMENTED** [review]
All looks good, still lots to do :-)

**Review by JAG-UK — APPROVED** [review]

File: draft-birkholz-scitt-scrapi.md
Comment (OR13) [reply draft-birkholz-scitt-scrapi.md:252]: YAML is growing in popularity, for cases where both humans and machines process the file, due to its readability and writability.

You could argue its a bad thing, if people start reading yaml instead of just using a machine to check it.

The point of the comment was to address requesting a representation of a receipt that a browser or filesystem might display nicely.

Comment (JAG-UK) [reply draft-birkholz-scitt-scrapi.md:252]: OK got you. It's more easily transmissible in a web/REST world and more supported by standard tooling than pure COSE/CBOR. Makes a lot of sense, our implementation wraps in JSON for the same reasons.

**Review by JAG-UK — APPROVED** [review]
LGTM

**Comment by OR13** [comment]
@henkbirkholz I can't merge this, but we should unless there are changes requested.

**Review by henkbirkholz — APPROVED** [review]

**Comment by henkbirkholz** [comment]
fixed collaborator list

**Pull request merged**
